package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nextcloud/text",
   "description": "Collaborative document editing",
-  "version": "26.0.0-alpha.7",
+  "version": "3.8.0",
```

Comment:
Needs double checking with @mejo- about the npm package that is still being built for collectives

Reply:
Yeah, better to keep it as is for now or bump it to `27.0.0-alpha.1`. `3.8.0` would be lower than the current version. And we need to do a new Text NPM release soon anyway in order to fix a few bugs in Collectives.

Reply:
Changed to `26.0.0-alpha.8` for now.